### DISCUSSION TITLE
Github pages deployment

### DISCUSSION THREAD

**Discussion by HarelSultan** [community/community#50044] [Other Feature Feedback, Questions, & Ideas] [answered]
Github pages deployment Bug

I'm trying  to deploy a single-page website i've made a while ago and i keep getting the following error :
404
File not found
The site configured at this address does not contain the requested file.

this is my repository link: https://github.com/HarelSultan/Meme-Generator

I made sure the index.html file exists and written correctly, and deployed it from the correct branch and still cant figure out what the problem is.

Anyone has run into the same problem or know how to fix it ? thank you

**Comment by Anmol-Baranwal** [comment, marked as answer]
I think you have mistakenly used the wrong name:
`inedx.html`, so kindly change it to `index.html`

This might resolve your issue. Feel free to ask me if the problem is not resolved.

**Reply by HarelSultan** [reply]
A true classic. Thank you!
